floodviz/hydrograph_utils.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+import operator
+import time
+
+
+def parse_hydrodata(jdata, newd, idx, sites_value_maxes):
+    siteName = jdata['value']['timeSeries'][0]['sourceInfo']['siteName']
+    key = siteName  # Key for this series
+    newd.append({'key': key, 'values': [], 'max': 0})
+    # Fill new data
+    max_val = 0
+    for idx2, obj in enumerate(jdata['value']['timeSeries'][0]['values'][0]['value']):
+        value = obj['value']
+        # For filtering series #
+        if max_val < float(value):
+            max_val = float(value)
+
+        dt = obj['dateTime']
+        date = dt.split('T')[0]
+        t = dt.split('T')[1].split('.')[0]
+        # reformat datetime for python datetime #   
+        dt = datetime.strptime(date + ' ' + t, '%Y-%m-%d %H:%M:%S')
+        # Convert to milliseconds for use with d3 x axis format
+        dt_ms = time.mktime(dt.timetuple()) * 1000
+        # (for below if statment) create dummy value for nvd3 issue at https://github.com/novus/nvd3/issues/695 #
+        if idx2 is 0: # First datapoint of this site
+            newd[idx]['values'].append({'date': date, "time": 0, "time_mili": dt_ms, 'value': 0, 'max': max_val})
+   
+        newd[idx]['values'].append({'date': date, "time": t, "time_mili": dt_ms, 'value': value, 'max': max_val})
```

Comment:
Was using for a .forceY call I was experimenting with in hydrograph.js but later removed.